raiden/ui/cli.py

```diff
@@ -174,12 +174,12 @@ def convert(self, value, param, ctx):
         type=int,
     ),
     click.option(
-        '--console/--no-console',
+        '--console',
```

Comment:
Since you changed it to a flag, I'd suggest a different help text:
`Start the command line interface`